README.md

```diff
@@ -72,7 +72,7 @@ Whether you're a one-time contributor or trying to join a community, working wit
 
 <a id="keep-requests-short-and-direct"></a>
 
-**Keep requests short and direct.** Much like sending an email, every contribution, no matter how simple or helpful, requires someone else's review. Many projects have more incoming requests than people available to help. Be concise. You will increase the chance that someone will be able to help you.
+**Keep requests short and direct.** Much like sending an email, every contribution, no matter how simple or helpful, requires someone else's review. Many projects have more incoming requests than people available to help. Be concise. You will increase the chances of someone helping you.
```

Comment:
1) More like the previous one, sounds more descriptive. And second, you can go suggest a fix in https://opensource.guide/ too ;d